frontend/lib/norent/homepage.tsx

```diff
@@ -1,19 +1,351 @@
+import { NorentLogo } from "./components/logo";
+
+type NorentImageType = "png" | "svg";
+
+export function getImageSrc(name: string, type?: NorentImageType) {
+  return `frontend/img/norent/${name}.${type || "svg"}`;
+}
+
+const checklistItems = [
+  "Build a letter using our free letter builder",
+  "Send your letter by email",
+  "Make a special request to your landlord",
+  "Send your letter by certified mail for free",
+  "Cite up-to-date legal ordinances in your letter",
+];
+
+const LandingPageChecklist = () => (
+  <div className="jf-space-below">
+    {checklistItems.map((checklistItem, i) => (
+      <article className="media" key={i}>
+        <div className="media-left">
+          <StaticImage
+            ratio="is-32x32"
+            src={getImageSrc("checkmark")}
+            alt="Check mark"
+          />
+        </div>
+        <div className="media-content">{checklistItem}</div>
+      </article>
+    ))}
+  </div>
+);
+
+const demandsListItems = ["Cancel rent", "Rent strike", "Ban evictions"];
+
+const LandingPageDemands = () => (
+  <div className="columns is-mobile is-multiline is-variable is-8">
+    {demandsListItems.map((demand, i) => (
+      <div className="column has-text-centered jf-has-centered-images" key={i}>
+        <p className="title is-size-5">{demand}</p>
+        <br />
+        <div className="jf-illustration-fist-pump">
+          <StaticImage
+            ratio="is-square"
+            src={getImageSrc("fistpump")}
+            alt="Fist pump"
+          />
+        </div>
+      </div>
+    ))}
+  </div>
+);
+
+const BuildMyLetterButton = () => (
+  <CenteredPrimaryButtonLink to={NorentRoutes.locale.letter.latestStep}>
+    Build my letter
+  </CenteredPrimaryButtonLink>
+);
 
 export const NorentHomepage: React.FC<{}> = () => (
-  <Page title="NoRent.org" withHeading="big" className="content">
-    <p>Hello, this is the no rent site.</p>
-    <p>Not much is here right now, but you can visit these pages:</p>
-    <ul>
-      <li>
-        <Link to={NorentRoutes.dev.styleGuide}>Style guide</Link>
-      </li>
-      <li>
-        <Link to={NorentRoutes.locale.letter.latestStep}>Letter builder</Link>
-      </li>
-    </ul>
+  <Page title="NoRent.org" className="content">
+    <section className="hero is-fullheight-with-navbar">
+      <div className="hero-body">
+        <div className="container jf-has-text-centered-desktop">
+          <h1 className="title is-spaced has-text-info">Can't pay rent?</h1>
+          <br />
+          <p className="subtitle">
+            You’re not alone. Millions of Americans won’t be able to pay rent
+            because of COVID-19. Use our FREE tool to take action by writing a
+            letter to your landlord.
+          </p>
+          <br />
+          <BuildMyLetterButton />
+          <br />
+        </div>
+      </div>
+      <div className="container jf-has-centered-images jf-space-below">
+        <Link to="#more-info" className="jf-jump-arrow">
```

Comment:
For some reason, this simple anchor link is not working at all for me. I guess it's a moot point since you will be investigating a new method for these kinds of links anyway.

Reply:
Yep, works great!